README.md

```diff
@@ -1,350 +1,57 @@
-# Ably Chat SDK for TypeScript and React
-
-<p style="text-align: left">
-    <img src="https://badgen.net/github/license/3scale/saas-operator" alt="License" />
-    <img src="https://img.shields.io/npm/v/@ably/chat.svg?style=flat">
```

Comment:
Having a shield like this can be useful to direct people to the NPM package and as a quick reference for the latest npm version

Reply:
We have a conversation planned, but I will re-add for now. Then we can edit what we need once we have decided what's coming or going!